facade/elrondNodeFacade.go

```diff
@@ -233,9 +235,9 @@ func (ef *ElrondNodeFacade) StatusMetrics() external.StatusMetricsHandler {
 	return ef.apiResolver.StatusMetrics()
 }
 
-// GetVmValue retrieves data from existing SC trie
-func (ef *ElrondNodeFacade) GetVmValue(address string, funcName string, argsBuff ...[]byte) ([]byte, error) {
-	return ef.apiResolver.GetVmValue(address, funcName, argsBuff...)
+// SimulateRunSmartContractFunction retrieves data from existing SC trie
+func (ef *ElrondNodeFacade) SimulateRunSmartContractFunction(command *smartContract.CommandRunFunction) (*vmcommon.VMOutput, error) {
```

Comment:
I liked the other input type better. you create a close connection between facade and smart contract processor.

Reply:
@camilbancioiu, @sasurobert, I'll keep `CommandRunFunction` inside `scDataGetter` and perform the following renaming:

- `scDataGetter` -> `SCQueryService`
- `CommandRunFunction` -> `SCQuery`
- `SimulateRunSmartContractFunction` -> `ExecuteQuery`